### ISSUE TITLE
Rests position when stem up forced

### ISSUE THREAD

**Issue by rigobauer** [paulrosen/abcjs#120]
When you force stems to go up with...
V:1 stem=up
... the rests go up too and are almost out of the staff. Rests shouldn't be affected by stem direction.

**Comment by paulrosen** [comment]
Thanks for the bug report: generally "stem=up" is paired with having two voices on the same line, so that is the desired behavior. I'll put in a test to see if it is the only voice on a line.

**Comment by rigobauer** [comment]
Yes, I supposed that was the reason. I first checked the abc standard to see if rest position was part of it, and it looks like not. Then I tried with three voices combining them with %%score and saw that. But with a single voice, looks strange. Check this case when you can...

X: 1
M: 4/4
L: 1/4
V:1 stem=up
EEEz

And in general, when you have a figure with a rest in the middle, I think it makes the horizontal bar go higher, and it also looks strange (like vertically too long)...

X: 1
M: 4/4
L: 1/8
V:1 stem=up
(3AzA

I hope it helps. And by the way, great plugin, man!

**Comment by paulrosen** [comment]
Fixed in 3.2.1

**Comment by rigobauer** [comment]
Rests are ok now, but something is happening with triplet indication. Line is not horizontal. Check the image...
![captura de pantalla 2017-11-30 a las 6 40 19](https://user-images.githubusercontent.com/29883162/33415802-6ab1b670-d599-11e7-95ed-2c1af3edff32.png)

**Comment by davidrthorn** [comment]
@rigobauer did you ever figure out how to render triplet brackets horizontal in all cases?

**Comment by rigobauer** [comment]
> @rigobauer did you ever figure out how to render triplet brackets horizontal in all cases?

No, sorry man. At that time I was in a hurry so I moved to another option that was more drum friendly. Anyway, a few weeks ago I revisited abcjs and it's incredible how much has been improved. Amazing work @paulrosen 

I'm just scheduling to find time for my personal side project (is where I need the drum notation), but I'm gonna move to abcjs for sure.
